egs/ami/s5b/local/chain/multi_condition/run_tdnn_lstm.sh

```diff
@@ -0,0 +1,334 @@
+#!/bin/bash
+
+# This is a chain-training script with TDNN+LSTM neural networks.
```

Comment:
Not checking this in detail as I assume it is similar to multi_condition/run_tdnn.sh except for the xconfig and some lstm specific parameters.